prime-router/src/main/kotlin/cli/tests/BasicTests.kt

```diff
@@ -66,172 +66,94 @@ class Ping : CoolTest() {
+         passed = passed and examinePostResponse(json, false)
+         if (!passed) {
+             bad("***end2end_up FAILED***: Error in post response")
+         }
+
+         val reportId = getReportIdFromResponse(json)
+             ?: return bad("***end2end_up FAILED***: Did not find report id")
+
+         // Need just a little wait for the db to catch up
+         waitABit(10, environment)
+
+         db = WorkflowEngine().db
+         db.transact { txn ->
+            expectedReceivers.forEach { expectedReceiver ->
+                // Check that reportId is attached to a created file and that the expected receiver matches actual
+                val fileName = sftpFilenameQuery(txn, reportId, expectedReceiver.name)
+                if (!fileName.isNullOrEmpty()) {
+                    good("The report was received by ${expectedReceiver.name}")
+                } else {
+                    bad("***end2end_up FAILED***: Did not find report id")
                     passed = false
                 }
-            }
-
-            // check translate step
-            val routeReportIds = getAllChildrenReportId(convertReportId)
-            if (routeReportIds.size < expectedReceivers.size) {
-                return bad(
-                    "***async end2end_up FAILED***: Expected at least ${expectedReceivers.size} route" +
-                        "report id(s), but got ${routeReportIds.size}."
-                )
-            }
-            routeReportIds.forEach { routeReportId ->
-                val translateResults = pollForStepResult(routeReportId, TaskAction.translate)
-                // verify each result is valid
-                for (result in translateResults.values)
-                    passed = passed && examineStepResponse(result, "translate", sender.topic)
-                if (!passed) {
-                    bad("***async end2end_up FAILED***: Translate result invalid")
-                }
 
-                if (sender.topic.isSendOriginal) { // original message pass through has translate directly call send
-                    // check send step
-                    val translateReportId = getSingleChildReportId(routeReportId)
-                        ?: return bad(
-                            "***async end2end_up FAILED***:" +
-                                " Did not find a translate report id from route: $routeReportId"
-                        )
-                    val sendResults = pollForStepResult(translateReportId, TaskAction.send)
-                    // verify each result is valid
-                    for (result in sendResults.values)
-                        passed = passed && examineStepResponse(result, "send", sender.topic)
-                    if (!passed) {
-                        bad("***async end2end_up FAILED***: Send result invalid")
-                    }
+                // Check that created file exists in sftp directory
+                val createdFile = File("./build/sftp/$fileName")
```

Comment:
So I'm actually not sure what this should be, but I don't know how this would work when running against the staging environment.  Probably a good 17th discussion...